language/move-model/src/model.rs

```diff
@@ -2673,7 +2682,13 @@ impl<'env> FunctionEnv<'env> {
         // We look up the `verify` pragma property first in this function, then in
         // the module, and finally fall back to the value specified by default_scope.
         let default = || match default_scope {
-            VerificationScope::Public => self.is_public(),
+            // TODO: by using `is_exposed`, we essentially mark all of Public, Script, Friend to be
```

Comment:
This makes sense to me--better to err on the side of verifying more, and we can always add more fine-grained options if needed. I would be ok with deleting the TODO.